tests/vmi_servers.go

```diff
@@ -32,6 +32,13 @@ func StartHTTPServer(vmi *v1.VirtualMachineInstance, port int) {
 	HTTPServer.Start(vmi, port)
 }
 
+func StartFedoraHttpServer(vmi *v1.VirtualMachineInstance, port int) {
```

Comment:
How about `StartPythonHttpServer`? If there is no such server on the VMI, be it Fedora or not, it will fail.

Reply:
Could you feedback on this?

Reply:
done.